docs/sites.yml

```
@@ -10931,3 +10931,36 @@
   built_by: Nagarjun Palavalli
   built_by_url: https://twitter.com/palavalli
   featured: false
+- title: Nyxo
+ url: https://www.nyxo.app
+ main_url: https://www.nyxo.app
+ description: >
+   Nyxo is the best aid for improving your sleep quality. We combine leading sleep research techniques with your sleep tracker’s data to provide you with personalized and actionable coaching, to improve your sleep.
+ categories:
+   - Healthcare
+   - Blog
+   - Science
+   - Technology
+   - Data
+   - Mobile Development
+ built_by: Kayla Gordon
+ built_by_url: https://www.kayla-gordon.com/
+ featured: false
+ - title: Earner
+ url: https://www.earner.ai/
+ main_url: https://www.earner.ai/
+ source_url: (optional - https://github.com/Earner-ai/earner-website
```

You've a broken bit here
```suggestion
 source_url: https://github.com/Earner-ai/earner-website
```

---

Just updated it. Thank you!